hokusai/production.yml

```diff
@@ -0,0 +1,133 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rosalind-web
+  namespace: default
+spec:
+  replicas: 2
```

Comment:
We should replace this with an auto-scaler, which will allow pods to scale up and down based on load.